go.mod

```diff
@@ -15,16 +15,15 @@ require (
 	github.com/docker/docker v1.4.2-0.20190910181529-415f8ecb65e8
 	github.com/docker/go-connections v0.4.0 // indirect
 	github.com/docker/go-units v0.4.0 // indirect
-	github.com/gogo/protobuf v1.3.0 // indirect
 	github.com/google/uuid v1.1.1
 	github.com/gorilla/mux v1.7.3 // indirect
 	github.com/hashicorp/go-cleanhttp v0.5.1 // indirect
 	github.com/hashicorp/go-getter v1.4.0
 	github.com/imdario/mergo v0.3.7
+	github.com/ipfs/testground/plans/dht v0.0.0-20191106194925-0d10fef12a43 // indirect
```

Comment:
These should not be necessary. I quite haven't identified why the go toolchain adds them some time. But you should try to remove them and do a `go build . && go mod tidy`, and they shouldn't reappear.

Reply:
Gotcha! Didn't know of `go mod tidy`